src/Ginibre.jl

```diff
@@ -30,30 +31,38 @@ struct Ginibre <: ContinuousMatrixDistribution
 end
 
 """
-    rand(W::Ginibre)
+    rand(rng::AbstractRNG, W::Ginibre)
```

Comment:
```suggestion
    rand([rng::AbstractRNG,] W::Ginibre)
```
see https://docs.julialang.org/en/v1/manual/documentation/.